static/js/alert.js

```diff
@@ -470,6 +483,27 @@ async function displayAlert(res){
         fetchLogsPanelData(data,-1).then((res)=>{
             alertChart(res);
         });
+        $('#query').val(res.queryParams.queryText);
+        $(`.ranges .inner-range #${res.queryParams.startTime}`).addClass('active');
+        datePickerHandler(res.queryParams.startTime, res.queryParams.endTime, res.queryParams.startTime)
+    } else if (res.alert_type === 3){
```

Comment:
Looks good.
nit: Instead of adding a third type (type 3), we should create a separate function to handle this, to avoid confusion later on.

We can keep the type as 2 and then create a separate function to display all the things, similar to what we have for logs, `createAlertFromLogs`.